packages/xo-server-sdn-controller/src/index.js

```diff
@@ -392,7 +392,7 @@ class SDNController extends EventEmitter {
         }' has been unplugged, electing a new host`
       )
       const newCenter = await this._electNewCenter(pif.$network, true)
-      poolNetwork.starCenter = newCenter ? newCenter.$ref : null
+      poolNetwork.starCenter = newCenter?.$ref
```

Comment:
Please do a separate PR for these unrelated changes.

Reply:
https://github.com/vatesfr/xen-orchestra/pull/4335